shared/teams/add-people/index.js

```diff
@@ -0,0 +1,144 @@
+// @flow
+import * as React from 'react'
+import {Box, Button, Dropdown, ProgressIndicator, Text, PopupDialog} from '../../common-adapters'
+import {globalStyles, globalMargins, globalColors} from '../../styles'
+import capitalize from 'lodash/capitalize'
+import {isMobile} from '../../constants/platform'
+import UserInput from '../../search/user-input/container'
+import SearchResultsList from '../../search/results-list/container'
+
+const MaybePopup = isMobile
```

Comment:
This is a good example of the lack of a solution for desktop and mobile pop up/modal split. 

We should think of a good abstraction that solves this for us. We have a ton of this kind of solution that’s similar but slightly different. It’s likely worth investing in this next sprint. 

Would love to hear people’s ideas. Maybe something route tree solves for us?